assets/css/dark-mode-preview.scss

```diff
@@ -38,7 +38,6 @@
 @import "./svg_paths";
 @import "./quiz";
 
```

Comment:
maybe this can be done when this file is edited

Reply:
Will take care of it from next time surely.